### PR TITLE
Add Components Documentation  Chinese translation

### PR DESCRIPTION
**Pull request by sss55b**
Hello, I have translated 9 documents into Chinese!

### CODE REVIEW

File: pages/content/amp-dev/documentation/components/reference/amp-date-picker-v0.1@zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,726 @@
...
+</amp-date-picker>
+[/sourcecode]
+
+### start-input-selector
+
+日期范围选择器的开始日期输入的查询选择器。如果省略此选项，则日期选择器会自动生成一个隐藏的输入字段，并为其指定名称 `start-date` 或 `${id}-start-date` 使用日期选择器的ID。如果这些冲突中的任何一个与表单中的现有元素冲突，都会发出错误。
+
+当 `amp-date-picker` 加载时，输入元素的值用于显示初始选择开始日期。
+
+通过[`src` attribute](#src-optional) 指定 `startDate` 属性来设置
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/components/reference/amp-date-picker-v0.1@zh_CN.md:229]: This link is broken and makes the check failing. Not your fault but already broken in the source document. This should actually simply link to `#src`.

**Review by matthiasrohmer — CHANGES_REQUESTED** [review]
Wow, thanks a lot! I can't say anything about the actual translation work, but this technically looks good except the one comment I added.

/cc @patrickkettner, @CrystalOnScript

File: pages/content/amp-dev/documentation/components/reference/amp-date-picker-v0.1@zh_CN.md (hunk 1 above)
Comment (sss55b) [reply pages/content/amp-dev/documentation/components/reference/amp-date-picker-v0.1@zh_CN.md]: done

**Review by matthiasrohmer — APPROVED** [review]
Thanks 🙂

**Pull request merged**